lib/armbian-configng/config.ng.software.sh

```diff
@@ -20,3 +20,95 @@ function see_monitoring() {
   fi
 }
+
+module_options+=(
+["update_skel,author"]="Kat Schwarz"
+["update_skel,ref_link"]=""
+["update_skel,feature"]="install_plexmediaserver"
+["update_skel,desc"]="Install plexmediaserver from repo using apt"
+["update_skel,example"]="install_plexmediaserver"
+["update_skel,status"]="Active"
+)
+#
+# Install plexmediaserver using apt
+#
+install_plexmediaserver(){
+if [ ! -f /etc/apt/sources.list.d/plexmediaserver.list ]; then
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/plexmediaserver.gpg] https://downloads.plex.tv/repo/deb public main" | sudo tee /etc/apt/sources.list.d/plexmediaserver.list > /dev/null 2>&1
+else
+    sed -i "/downloads.plex.tv/s/^#//g" /etc/apt/sources.list.d/plexmediaserver.list > /dev/null 2>&1
+fi
+# Note: for compatibility with existing source file in some builds format must be gpg not asc
+# and location must be /usr/share/keyrings
+wget -qO- https://downloads.plex.tv/plex-keys/PlexSign.key | gpg --dearmor | sudo tee /usr/share/keyrings/plexmediaserver.gpg > /dev/null 2>&1
+debconf-apt-progress -- apt-get update
+debconf-apt-progress -- apt-get -y install plexmediaserver
+whiptail --msgbox "To test that Plex Media Server  has installed successfully\nIn a web browser go to http://localhost:32400/web or \nhttp://127.0.0.1:32400/web on this computer." 9 70
+}
+
+module_options+=(
+["update_skel,author"]="Kat Schwarz"
+["update_skel,ref_link"]=""
+["update_skel,feature"]="install_embyserver"
+["update_skel,desc"]="Download a embyserver deb file from a URL and install using apt"
+["update_skel,example"]="install_embyserver"
+["update_skel,status"]="Active"
+)
+#
+# Download a deb file from a URL and install using wget and apt with dialog progress bars
+#
+install_embyserver(){
+    URL=$(curl -s https://api.github.com/repos/MediaBrowser/Emby.Releases/releases/latest | \
+    grep "/emby-server-deb.*$(dpkg --print-architecture).deb" | cut -d : -f 2,3 | tr -d '"')
+    cd ~/
+    wget -O "emby-server.deb" $URL 2>&1 | stdbuf -oL awk '/[.] +[0-9][0-9]?[0-9]?%/ { print substr($0,63,3) }' | \
+    whiptail --gauge "Please wait\nDownloading ${URL##*/}" 8 70 0
+    debconf-apt-progress -- apt-get -y install ~/emby-server.deb
+    unlink emby-server.deb
+    whiptail --msgbox "To test that Emby Server  has installed successfully\nIn a web browser go to http://localhost:8096 or \nhttp://127.0.0.1:8096 on this computer." 9 70
+}
+
+module_options+=(
+["update_skel,author"]="Kat Schwarz"
+["update_skel,ref_link"]=""
+["update_skel,feature"]="install_docker"
+["update_skel,desc"]="Install docker from a repo using apt"
+["update_skel,example"]="install_docker engine"
+["update_skel,status"]="Active"
+)
+#
+# Install Docker from repo using apt
+# Setup sources list and GPG key then install the app. If you want a full desktop then $1=desktop
+#
+install_docker(){
+# Check if repo for distribution exists.
+dist_id=$(cat /etc/*release | grep "ID=ubuntu\|ID=debian" | cut -c 4-)
```

Comment:
We read this already when config starts.

Reply:
Found $DISTRO with capitalized format (Debian/Ubuntu).
I need debian/ubuntu.
Used distro=$(echo $DISTRO | tr '[:upper:]' '[:lower:]').
Is there a better way?